README.md

```diff
@@ -23,14 +23,17 @@ ExpMan provides a set of handy commands that not only allows beginners to quickl
 **Currently Only Zsh is supported**
 
 1. Install [fzf][fzf]
-2. Download
-```bash
-git clone https://github.com/megvii-research/expman $HOME/.expman
-```
-3. Load expman by adding one line in `~/.zshrc`
-```bash
-echo '[[ -o interactive ]] && source $HOME/.expman/expman.zsh' >> ~/.zshrc
-```
+    ```bash
+    brew install fzf
+    ```
```

Comment:
I think it's better to keep blank here. The correct method of installation depends on user's platform. It's difficult to cover. Fzf's doc already include various installation methods.

Reply:
Done. PTAL @bigeagle

Reply:
LGTM 👍